src/modules/image.cpp

```diff
@@ -24,6 +25,18 @@ std::unordered_map<ImageCategory, std::string> imageCategoryToLoremFlickrStringM
     {ImageCategory::Fashion, "fashion"}, {ImageCategory::People, "people"},     {ImageCategory::Nature, "nature"},
     {ImageCategory::Sports, "sports"},   {ImageCategory::Technics, "technics"}, {ImageCategory::Transport, "transport"},
 };
+
+int randomInt(int min, int max) {
+    return number::integer(min, max);
+}
```

Comment:
please remove that function and ue number::integer directly in below randomColor function